### ISSUE TITLE
Update Kafka consumer to support SASL authentication

### ISSUE THREAD

**Issue by vsnine** [F5Networks/f5-telemetry-streaming#10]
- Feature Idea

##### Telemetry Streaming BUILD/ VERSION

```
f5-telemetry-1.2.0-1.noarch.rpm
```

##### BIGIP VERSION

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.1
  Build       0.0.4
  Edition     Final
  Date        Fri Jul 20 17:55:49 PDT 2018
```

##### SUMMARY

```
Kafka consumer appears to not support SASL authentication.
```

##### STEPS TO REPRODUCE

```
The underlying kafka-node library appears to support username and password but it does not appear to have been implemented in the consumer module in f5telemetry.
```

##### ACTUAL RESULTS

```

```

**Comment by jsevedge** [comment]
@vsnine Internal ID 149 opened to track this.

**Comment by vsnine** [comment]
Small snag, the version of kafka-node that appears to be bundled in the 1.2.0 release is 2.6.1. SASL appears to have not been implemented until the 3.0.0 release of kafka-node. https://github.com/SOHU-Co/kafka-node/pull/923

**Comment by focrensh** [comment]
TS version 1.3 introduced the support of SASL-PLAIN auth for Kafka. For further action please open a separate issue.
